File: java/PlanesAndroid/app/src/main/java/com/planes/common/PlayerGuessReaction.java
package com.planes.common;

public class PlayerGuessReaction {

    //victory for computer or player
    public boolean m_RoundEnds = false;
    //is it a draw
    public boolean m_IsDraw = false;
    //who won
    public boolean m_isPlayerWinner = false;
    //if no victory then a computer move is generated
    public boolean m_ComputerMoveGenerated = false;
    //which computer move was generated
    public GuessPoint m_ComputerGuess;
    public GameStatistics m_GameStats;

    public PlayerGuessReaction() {
        m_ComputerGuess = new GuessPoint(0, 0);
        m_GameStats = new GameStatistics();
    }

}

File: kotlin/PlanesAndroid/app/src/main/java/com/planes/common/PlayerGuessReaction.kt
package com.planes.common

import com.planes.common.GuessPoint
import com.planes.common.GameStatistics

class PlayerGuessReaction {
    //victory for computer or player
    var m_RoundEnds = false

    //is it a draw
    var m_IsDraw = false

    //who won
    var m_isPlayerWinner = false

    //if no victory then a computer move is generated
    var m_ComputerMoveGenerated = false

    //which computer move was generated
    var m_ComputerGuess: GuessPoint

    var m_GameStats: GameStatistics

    init {
        m_ComputerGuess = GuessPoint(0, 0)
        m_GameStats = GameStatistics()
    }
}